### PR TITLE
:arrow_up: bump k-ramel version

### PR DESCRIPTION
**Pull request by bpetetot**
- Needs k-ramel release with PR https://github.com/alakarteio/k-ramel/pull/198

### CODE REVIEW

File: src/screens/organizer/proposals/proposals.reactions.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import { reaction } from 'k-ramel'
+
+import { flow, isEqual, omit, over, pick, update, pickBy } from 'lodash/fp'
+
+import * as firebase from 'firebase/proposals'
+
+/* set proposal filters from URL query params */
+export const setProposalFiltersFromRouter = reaction((action, store, { router }) => {
```
Comment (fabienjuif) [inline src/screens/organizer/proposals/proposals.reactions.js]: You can remove the `reaction` wrapper.

This is needed when you want to do: 
```js
const setProposalFiltersFromRouter = reaction((action, store, drivers) => { })
//...
const listeners = [
  setProposalFiltersFromRouter.when('@@trigger'),
  setProposalFiltersFromRouter.when('@@trigger_2')
]
```

But in your case this is useless and bloated :)

File: src/store/reducers/data/index.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 export default {
-  events: keyValue({ key: 'id' }),
-  organizations: keyValue({ key: 'id' }),
-  proposals: keyValue({ key: 'id' }),
-  ratings: keyValue({ key: 'uid' }),
-  talks: keyValue({ key: 'id' }),
-  users: keyValue({ key: 'uid' }),
-  surveys: keyValue({ key: 'uid' }),
+  events: types.keyValue({ key: 'id' }),
```
Comment (fabienjuif) [inline src/store/reducers/data/index.js]: If you want, you can remove: `{ key: 'id' }` since `id` is the new default key

**Comment by fabienjuif** [comment]
If you want me to try migrating `conference-hall` to `k-redux-router` and `@@k-ramel/driver-form` feel free to open issues and assign them to me ;)

File: src/screens/organizer/proposals/proposals.reactions.js (hunk 1 above)
Comment (bpetetot) [reply src/screens/organizer/proposals/proposals.reactions.js]: Great! thx

File: src/store/reducers/data/index.js (hunk 2 above)
Comment (bpetetot) [reply src/store/reducers/data/index.js]: Super great! ;)

**Pull request merged**